remote_branch_checker/remote_branch_checker.sh

```diff
@@ -249,6 +249,13 @@ ${gitcmd} merge -q --no-edit FETCH_HEAD
 exitstatus=${PIPESTATUS[0]}
 if [[ ${exitstatus} -ne 0 ]]; then
     echo "Error: The ${branch} branch at ${remote} does not apply clean to ${baseref}" | tee -a ${errorfile}
+
+    mergeconflicts="$( ${gitcmd} diff --name-only --diff-filter=U )"
```

Comment:
Tiny comment here... maybe we also need to add the `--relative` option to avoid any full path to be printed?

Reply:
I don't think we need `--relative` as this is for when we run from a project subdirectory. With `--relative`, it can be said to exclude changes outside the directory and show pathnames relative to it. All other file paths are always relative to the git project. I can't recall ever seeing absolute file paths with any git command output.